packages/atomic/src/components/commerce/atomic-commerce-recommendation-list/atomic-commerce-recommendation-list.spec.ts

```diff
@@ -0,0 +1,834 @@
+import {fixtureCleanup} from '@/vitest-utils/testing-helpers/fixture-wrapper.js';
+import {
+  defaultBindings,
+  renderInAtomicCommerceRecommendationInterface,
+} from '@/vitest-utils/testing-helpers/fixtures/atomic/commerce/atomic-commerce-recommendation-interface-fixture.js';
+import {buildFakeProduct} from '@/vitest-utils/testing-helpers/fixtures/headless/commerce/product.js';
+import {buildFakeRecommendations} from '@/vitest-utils/testing-helpers/fixtures/headless/commerce/recommendations-controller.js';
+import {buildFakeSummary} from '@/vitest-utils/testing-helpers/fixtures/headless/commerce/summary-subcontroller.js';
+import {
+  buildRecommendations,
+  InteractiveProduct,
+  InteractiveProductProps,
+  Product,
+} from '@coveo/headless/commerce';
+import {page} from '@vitest/browser/context';
+import '@vitest/browser/matchers.d.ts';
+import {html} from 'lit';
+import {describe, expect, vi, beforeEach, it} from 'vitest';
+import {
+  ItemDisplayBasicLayout,
+  ItemDisplayDensity,
+  ItemDisplayImageSize,
+} from '../../common/layout/display-options.js';
+// TODO: Replace with atomic-commerce-recommendation-interface bindings once it is merged (KIT-3934)
+import {CommerceBindings} from '../atomic-commerce-interface/atomic-commerce-interface.js';
+import './atomic-commerce-recommendation-list';
+import {AtomicCommerceRecommendationList} from './atomic-commerce-recommendation-list.js';
+
+vi.mock('@/src/components/common/interface/store', {spy: true});
+vi.mock('@coveo/headless/commerce', {spy: true});
+
+describe('AtomicCommerceRecommendationList', () => {
+  const interactiveProduct = vi.fn();
+  const promoteChildToParent = vi.fn();
+  const summary = vi.fn();
+
+  beforeEach(() => {
+    fixtureCleanup();
```

Comment:
I don't think this is needed; fixtureCleanup is called automatically after each test.